Fika.Core/UI/Custom/MatchMakerUIScript.cs

```diff
@@ -428,21 +352,6 @@ private void RefreshUI()
                         }
                         break;
                     case LobbyEntry.ELobbyStatus.IN_GAME:
-                        tooltipTextGetter = new()
```

Comment:
This should be re-added as it is not supported and will cause too many side effects that we don't want to bother with until officially supported.